.buildkite/scripts/run_models.sh

```diff
@@ -21,4 +21,6 @@ dbt run --target "$db" --full-refresh
 dbt test --target "$db"
 dbt run --vars '{ad_reporting__url_report__using_null_filter: false}' --target "$db" --full-refresh
 dbt test --vars '{ad_reporting__url_report__using_null_filter: false}' --target "$db"
+dbt run --vars '{tiktok_ads__using_ad_country_report: false, tiktok_ads__using_campaign_country_report: false}' --target "$db" --full-refresh
```

Comment:
Since you're no longer bringing in `ad_country_report`, the corresponding variable isn't needed.   

```suggestion
dbt run --vars '{tiktok_ads__using_campaign_country_report: false}' --target "$db" --full-refresh
```